lib/Bio/EnsEMBL/DataCheck/Checks/MartDatasetName.pm

```diff
@@ -51,7 +51,7 @@ sub skip_tests {
   my $production_name = $mca->get_production_name;
   my $schema_version = $mca->get_schema_version;
 
-  if ($division =~ '/EnsemblVertebrates|EnsemblMetazoa|EnsemblPlants/') {
+  if ($division =~ '/EnsemblVertebrates/') {
```

Comment:
I made this change so that the `MartDatasetName` datacheck would be consistent with the ensembl-biomart repo, which (as of [commit 2850347](https://github.com/Ensembl/ensembl-biomart/commit/2850347631704c9ab8a72c465dbbf5c2caac78e4)) does not use either the Plants or Metazoa Compara `allowed_species.json` file for species filtering.